src/domain/Movies/Catalog/Search.js

```diff
@@ -0,0 +1,8 @@
+import { searchMoviesInAPI } from "../../../infra/Movies/Catalog/Search"
+import { formatedMoviesFromAPI } from "../../Utils/formatedMovies"
+
+export const searchMovies = async (search) => {
+    const movies = await searchMoviesInAPI(search);
+    const formatedMovies = await formatedMoviesFromAPI(movies);
+    return formatedMovies;
+} 
```

Comment:
penses à installer ESlint pour  avoir la blank line en fin de fichier automatique :) 
pour l'anecdote dans ma boite la première PR que j'ai fais, j'ai eu 15 retours avec simplement écrit "EOF", va comprendre ce que ça voulait dire xD (spoiler alert:  End Of File)

Reply:
Génial l'anecdote 🤣 

Je vais l'installer !